extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -1,37 +1,849 @@
+- Locality of reference: Adjacent indices are stored close to each other in memory and are close to each other spatially.
+
+#### Morton Order Example
+
+_The following section is non-normative_
+
+The figure below shows the tile location decomposition of the tile `(level, x, y) = (3, 5, 1)`. We first convert the tile location to its Morton index, which is `19`. At Level 3 of a Quadtree, we'll use 6 bits to represent the binary value of the Morton index: `010011`.
+
+![Morton Order](figures/morton-indexing.png)
+
+### Availability Encoding
+
+Availability bitstreams are packed in binary using the format described in the [Boolean Data section](https://github.com/CesiumGS/3d-tiles/blob/3d-tiles-next/specification/Metadata/0.0.0/README.md#boolean-data) of the Cesium 3D Metadata Specification. These bitstreams are subject to alignment requirements described in the [Binary Alignment Rules section](https://github.com/CesiumGS/3d-tiles/tree/3DTILES_binary_buffers/extensions/3DTILES_binary_buffers) of the same specification.
+
+Each availability bitstream must be stored as a separate `bufferView`, but multiple `bufferViews` may be stored in a single `buffer`.
+
+## Tileset JSON
+
+Using implicit tiling, the tileset JSON file describes the root tile, tiling scheme, and template URIs for locating resources.
+
+In the extension object, the following information about the root tile is included:
+
+| Property | Description |
+| ------ | ----------- |
+| `tilingScheme` | Either `QUADTREE` or `OCTREE`|
+| `boundingVolume` | a bounding volume (either a `box` or `region`) describing the root tile |
+| `refine` | Either `ADD` or `REPLACE` as in the Cesium 3D Tiles 1.0 Specification. |
+| `geometricError` | Geometric error of the root tile as described in the Cesium 3D Tiles 1.0 Specification. |
+| `maximumLevel` | Maximum level of the entire tree |
+| `subtreeLevels` | How many distance levels in each subtree. |
+
+Furthermore, template URIs are used for locating subtree JSON files as well as tile contents. The key properties are as follows:
+
+| Option | Description |
+| ------ | ----------- |
+| `subtrees.uri` | template URI for a subtree JSON file. see [Subtrees](#subtrees) for more info |
+| `content.uri` | template URI for the content 3D Models |
+
+Below is a full example of how the tileset JSON file looks in practice:
+
+```json
+{
+  "asset": {
+    "version": "1.0"
+  },
+  "geometricError": 10000,
+  "extensionsUsed": [
+    "3DTILES_implicit_tiling",
+  ],
+  "extensionsRequired": [
+    "3DTILES_implicit_tiling",
+  ],
+  "extensions": {
+    "3DTILES_implicit_tiling": {
+      "extras": {
+        "draftVersion": "0.0.0"
+      },
+      "boundingVolume": {
+        "region": [-1.318, 0.697, -1.319, 0.698, 0, 20]
+      },
+      "refine": "REPLACE",
+      "tilingScheme": "QUADTREE",
+      "geometricError": 5000,
+      "subtreeLevels": 7,
+      "maximumLevel": 21,
```

Comment:
While it's totally valid for `maximumLevel` to be 20 or 21, I think the example should be 20 so that three subtrees neatly fit into the tileset JSON. With 21 there is spillover into a fourth layer of subtrees.